### DISCUSSION TITLE
Question: Red exclamation next to app

### DISCUSSION THREAD

**Discussion by lividhen** [seedvault-app/seedvault#215] [Q&A]
I am using the Crdroid 6 implementation of Seedvault, so i don't know whether to ask here or on the CrDroid github.
What is this red exclamation? It does not give me any other information on it... It was next to Google Maps, Stardew Valley, and Tasker, then I hit backup again and the one next to Stardew Valley disappeared, and now there is just one next to maps and Tasker. What does it mean?
![image](https://user-images.githubusercontent.com/42221797/109203024-e5267280-7760-11eb-9994-6c9f9f9c8fb1.png)

**Comment by grote** [comment]
It means that the app couldn't get backed up. As regards to Google Maps this is a duplicate of #147

Note there's https://github.com/seedvault-app/seedvault/discussions for questions. This is for tickets.
